app.js

```diff
@@ -13,13 +14,34 @@ app.set('views', path.join(__dirname, 'views'));
 app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
-
+hbs.registerPartials(__dirname + '/views/partials');
 // ...
 
 // Add the route handlers here:
-
 app.get('/', (req, res) => {
-  res.render('index');
+  res.render('index.hbs');
+});
+
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
+      res.render('beers.hbs', { beersFromApi });
+      console.log('Beers from the database: ', beersFromApi);
+      console.log(beersFromApi.length);
+    })
+    .catch(error => console.log(error));
 });
 
-app.listen(3000, () => console.log('🏃‍ on port 3000'));
+app.get('/random-beer', (req, res) => {
+  punkAPI
+    .getRandom()
+    .then(randomBeer => {
+      res.render('random-beer.hbs', { randomBeer });
```

Comment:
Thanks to line 12 `app.set('view engine', 'hbs');`, Express is able to add the extension `.hbs` by itself when using `res.render()`.

So you don't need to add it, you can refer to the file name only and it'll work the same
```
res.render('random-beer', { randomBeer });